Index: assets/in
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- assets/in	(revision 225ebf1e7398a518344870ecf87a8536a6bddb7b)
+++ assets/in	(revision 8d6c29503b6d41c2e3f4c09de5e710cacb828ac1)
@@ -112,6 +112,8 @@
   git branch $branch FETCH_HEAD
 done
 
+glide install --ignore-platform-reqs --no-interaction --optimize-autoloader
+
 if [ "$ref" == "HEAD" ]; then
   return_ref=$(git rev-parse HEAD)
 else
Index: assets/common.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- assets/common.sh	(revision 8d6c29503b6d41c2e3f4c09de5e710cacb828ac1)
+++ assets/common.sh	(revision 201ee265bb26457ca869fc55bf5c35a49052fc54)
@@ -99,13 +99,22 @@
   ]"
 }
 
+add_glide_metadata() {
+  local version=$(glide --version | jq -s -R .)
+
+  jq ". + [
+    {name: \"glide-version\", value: ${version}}
+  ]"
+}
+
 git_metadata() {
   jq -n "[]" | \
     add_git_metadata_basic | \
     add_git_metadata_committer | \
     add_git_metadata_branch | \
     add_git_metadata_tags | \
-    add_git_metadata_message
+    add_git_metadata_message | \
+    add_glide_metadata
 }
 
 configure_credentials() {
